### DISCUSSION TITLE
请作者帮忙

### DISCUSSION THREAD

**Discussion by TotoWang-hhh** [DearVa/ExplorerEx#37] [Q&A] [answered]
如果我没记错的话，您曾经给该软件开发过一个预览文件的功能，请问能否把音频预览的那部分弄成exe借我用用。我最近需要这种音频预览程序，但是目前使用的界面库写不出来

**Comment by DearVa** [comment, marked as answer]
你可以试试[QuickLook](https://github.com/QL-Win/QuickLook)

**Reply by TotoWang-hhh** [reply]
主要不是自用，我需要让我写的东西能够听音频，然后这个播放器不能太复杂……
简单说下理想中的啥样子：能拉进度能暂停，没有切歌和播放列表

**Reply by TotoWang-hhh** [reply]
已找其他人解决，谢谢回复
